app/services/npq/void_participant_outcome.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module NPQ
+  class VoidParticipantOutcome
+    def initialize(participant_declaration)
+      @participant_declaration = participant_declaration
+    end
+
+    def call
+      return unless create_participant_outcome?
+      return if latest_participant_outcome&.voided?
+
+      ParticipantOutcome::NPQ.create!(
```

Comment:
could we call `CreateParticipantOutcome` instead as it has further validations?

Reply:
@cwrw 
My initial implementation was to use `CreateParticipantOutcome`, but that has extra validation we don't need for void. Thought it would be better to create a simpler service.